music21/metadata/testMetadata.py

```diff
@@ -15,6 +18,27 @@ def testMetadataLoadCorpus(self):
         c = converter.parse(mTF.mozartTrioK581Excerpt)
         md = c.metadata
 
+        self.assertEqual(md['movementNumber'], (metadata.Text('3'),))
+        self.assertEqual(md['movementName'],
+            (metadata.Text('Menuetto (Excerpt from Second Trio)'),))
```

Comment:
indents.  Make the two things being compared as equal be at the same place.  With multiple lines close the parentheses on a new line.  Especially important for lines ending `)'),))`

Reply:
That's a heck of a line ending.  I hadn't even noticed that.  I'll fix.

Reply:
Done.